apps/react-storybook/stories/chat/Chat.stories.tsx

```diff
@@ -390,3 +390,47 @@ export const Customization: Story = {
         );
     }
 }
+
+export const TypingUsers: Story = {
+    args: {
+        width: 400,
+        height: 500,
+        typingUsers: [],
+    },
+    argTypes: {
+        typingUsers: {
+            control: 'select',
+            defaultValue: 'No one is typing',
+            options: [
+                'No one is typing',
+                'One user typing',
+                'Two users typing',
+                'Three users typing',
+            ],
+            mapping: {
+                ['No one is typing']: [],
+                ['One user typing']: [ firstAuthor ],
+                ['Two users typing']: [ firstAuthor, secondAuthor ],
+                ['Three users typing']: [ firstAuthor, secondAuthor, thirdAuthor ],
```

Comment:
You forgot to demonstrate scenario Multiple Users are typing:  "{0} and others are typing..."

Reply:
e71c9b02ce630f5a790eacc297e696b421101a5e